src/misc/fs_utils_win32.cpp

```diff
@@ -256,9 +256,49 @@ void set_dos_file_time(const NativeFileHandle handle, const uint16_t date,
 	SetFileTime(handle, nullptr, nullptr, &write_time);
 }
 
+// Includes a Windows specific hack
+// We move the file to be deleted to a temp file before delete
+// This allows a file with the same name to be created even if there are open file handles
+// See bug report about the game Abuse: https://github.com/dosbox-staging/dosbox-staging/issues/4123
 bool delete_native_file(const std_fs::path& path)
 {
-	return remove(path.string().c_str()) == 0;
+	// Prefix only uses first 3 letters, abbreviate DosBox Staging
+	const wchar_t *prefix = L"DBS";
+
+	// Zero for unique_number uses system time
+	// Zero also means it creates the file and ensures it is unique
+	constexpr UINT unique_number = 0;
+
+	// Buffer size is Win32 defined MAX_PATH plus a null terminator
+	wchar_t temp_file[MAX_PATH + 1] = {};
+
+	// Create a temporary file inside the same directory as the file to be deleted
+	// Returns unique identifier on success and 0 on failure
+	if (GetTempFileNameW(path.parent_path().c_str(), prefix, unique_number, temp_file) == 0) {
+		LOG_WARNING("FS: Failed to create temp file. Deleting '%s' directly.", path.string().c_str());
```

Comment:
Should rather be a `LOG_ERR`, no?

Reply:
Sure, that makes sense.  Changed to `LOG_ERR`